src/error.h

```diff
@@ -26,37 +26,27 @@
 #define HSK_EUNEXPECTEDNODE 17
 #define HSK_ERECORDMISMATCH 18
 
-// Cuckoo
-#define HSK_EPOWOK 0
-#define HSK_EPOWPROOFSIZE 19
-#define HSK_EPOWTOOBIG 20
-#define HSK_EPOWTOOSMALL 21
-#define HSK_EPOWNONMATCHING 22
-#define HSK_EPOWBRANCH 23
-#define HSK_EPOWDEADEND 24
-#define HSK_EPOWSHORTCYCLE 25
-
 // POW
-#define HSK_ENEGTARGET 26
-#define HSK_EHIGHHASH 27
+#define HSK_ENEGTARGET 19
```

Comment:
Just curious, are there any side effects for changing the error numbers here?

Reply:
The integers defined in this file match up with the error message strings in `src/error.c`, see https://github.com/handshake-org/hnsd/pull/24/files#diff-ea12193fb130aea2da140db9c638871bR45